src/app/components/checkout/checkout.component.ts

```diff
@@ -49,7 +49,9 @@ export class CheckoutComponent implements OnInit {
     this.carts$.subscribe((items) => {
       let total = 0;
       for (const item of items) {
-        total += item.price * item.orderedUnit;
+        if(item.paymentOption === 'online'){
+          total += item.price * item.orderedUnit;
```

Comment:
We need to calculate the total even if it's not online , 
What we would do is to extract items from the cart that are online payment option then calculate the sum and send to paystack

Reply:
fixed